stock_picking_batch_oca/models/stock_batch_picking.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+
+from odoo.exceptions import UserError
+
+
+class StockBatchPicking(models.Model):
+    """ This object allow to manage multiple stock.picking at the same time.
+    """
+    # renamed stock.batch.picking -> stock.picking.batch
+    _inherit = 'stock.picking.batch'
+
+    name = fields.Char(
+        string='Name',
```

Comment:
Should we overwrite the standard user's label? Original is 'Batch Picking Name' https://github.com/odoo/odoo/blob/12.0/addons/stock_picking_batch/models/stock_picking_batch.py#L15